products/veeam-backup-and-replication

```diff
@@ -0,0 +1,123 @@
+---
+title: Veeam Backup & Replication
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backup-and-replication
+alternate_urls:
+-   /veeam-backup
+releasePolicyLink: https://www.veeam.com/product-lifecycle.html
+activeSupportColumn: End of Fix
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of support
+
+releases:
+-   releaseCycle: "12.0"
```

Comment:
```suggestion
-   releaseCycle: "12"
```

It seems they do not use the minor version anymore (source https://www.veeam.com/kb2680).